ores/features_reverted.py

```diff
@@ -61,8 +61,10 @@ def main():
         rev_pages = read_rev_pages(sys.stdin)
     else:
         rev_pages = read_rev_pages(open(args['--rev-pages']))
-
-    features = import_from_path(args['<features>'])
+    if '<features>' in args:
```

Comment:
Currently if <features> does not exist we display help, this behavior would need to be changed if we want to go in this direction. We can also define a keyword like "None" so that it takes the "generic.generic" features. The piece of code has no impact in functionality currently.